Running `gulp test --files` from a clean state is broken due to missing CSS

#10798 added a dependency on `../build/amp-ad-0.1.css.js` from `testing/describes.js`. However, it did not ensure that `../build/amp-ad-0.1.css.js` is always generated when `gulp test --files` is invoked from a clean state.

It needs to ensure that `testing/describes.js` is included in the css build even when `gulp test` is called with `--files`.

As a result, the following workflow is broken...
```
gulp clean
gulp test --unit --files <test file>
```
... and results in the following error:
```
09 10 2017 16:07:54.040:ERROR [framework.browserify]: bundle error
09 10 2017 16:07:54.041:ERROR [framework.browserify]: Error: Cannot find module '../build/amp-ad-0.1.css.js' from '/usr/local/google/home/rsimha/src/amphtml/testing'
Chrome 61.0.3163 (Linux 0.0.0) ERROR
  Uncaught Error: bundle error (see logs)
  at /tmp/0b8b9f3183f8912dbe5da828f1b908cb.browserify:1
```

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -412,7 +412,7 @@ function compileCss() {
   })
   .then(() => {
     return buildExtensions({
-      bundleOnlyIfListedInFiles: true,
+      bundleOnlyIfListedInFiles: false,
       compileOnlyCss: true
     });
   });

